### ISSUE TITLE
Event date in the question is not required, but our agents needs it

### ISSUE THREAD

**Issue by kongzii** [gnosis/prediction-market-agent-tooling#481]
[From the rules](https://cdn.kleros.link/ipfs/QmZM12kkguXFk2C94ykrKpambt4iUVKsVsxGxDEdLS68ws/omen-rules.pdf):

> If no specific event date is given in a question, it is assumed that the resolution date will be considered the event date.

Currently, we filter out questions without event date in them here https://github.com/gnosis/prediction-market-agent-tooling/blob/main/prediction_market_agent_tooling/tools/is_predictable.py#L16.

And then we don't provide market's closing time into LLM.

So, the current implementation works fine, but it could skip some markets unnecessarily.

So if want to stick to the rules, we should:

1. Don't require event date in the question itself.
2. Feed market's closing date to agents.

It sounds like a straightforward and fast thing to do, but I'm worried it could have some accuracy/profitability consequences.

Because if question contains event date but we also provide closing date to the agent, it could be confused about it and predict the event according to the wrong date.  

We could do some if/elsing and use the closing date only if the question doesn't contain it already, because otherwise, closing date is theoretically irrelevant. 

Practically, if closing date is too much in the future, agent will have the funds locked in there.

**Comment by greenlucid** [comment]
Note that the rules on the new default Arbitrator, [31 Jurors that was setup here](https://github.com/gnosis/prediction-market-agent-tooling/blob/8bbcd0caaa09a0958889c20eb4e8d3a445546ad1/prediction_market_agent_tooling/markets/omen/omen_contracts.py#L538), are different from the ones you referenced in your issue. So, if they're different (they're pretty similar but there might be differences?) those two need to be juggled simultaneously?

https://ipfs.kleros.io/ipfs/QmaUr6hnSVxYD899xdcn2GUVtXVjXoSXKZbce3zFtGWw4H/Question_Resolution_Policy.pdf

**Comment by kongzii** [comment]
Good point, thanks! Yeah, it needs to be checked when this issue is implemented. Worst-case, we will need to accept the Arbitrator enum and make differences based on that.
